lib/Net/SSLeay.pod

```diff
@@ -8886,6 +8886,40 @@ like X509_set_pubkey.
 This function has no equivalent in OpenSSL but combines multiple OpenSSL
 functions for an easier interface.
 
+=item * CTX_set_ecdh_auto, set_ecdh_auto
+
+These functions enable or disable the automatic curve selection on the server
+side by calling SSL_CTX_set_ecdh_auto resp. SSL_set_ecdh_auto.
+If enabled the highest preference curve is automatically used for ECDH temporary
+keys used during key exchange.
+This function is no longer available for OpenSSL 1.1.0 or higher.
+
+  Net::SSLeay::CTX_set_ecdh_auto($ctx,1);
+  Net::SSLeay::set_ecdh_auto($ssl,1);
+
+=item * CTX_set1_curves_list, set1_curves_list
+
+These functions set the supported curves in the order of preferences by calling
```

Comment:
```suggestion
These functions set a preferred order for the supported curves by calling
```

Reply:
This edit suggests that in only sets the preferred order for the curves. But it actually sets both which curves will be supported and als the preference order.

Reply:
Thanks for the clarification - how about "These functions set the list of supported curves (in order of preference) by calling"?

Reply:
I'm not really sure how this improves the understanding but I'm not a native English speaker. But since it is not wrong I'm not insisting on my wording.